satpy/composites/__init__.py

```diff
@@ -589,6 +589,17 @@ def __call__(self, projectables, optional_datasets=None, **info):
 class NIRReflectance(CompositeBase):
     """Get the reflective part of NIR bands."""
 
+    def __init__(self, **kwargs):
+        """Collect custom configuration values.
+
+        Args:
+            max_sza (float): Maximum solar zenith angle in degrees that is
+                considered valid and correctable. Default 95.0.
+
+        """
+        self.sunz_threshold = kwargs.get('sunz_threshold')
```

Comment:
```suggestion
        self.sunz_threshold = max_sza
```

Reply:
oh and there seems to be a missmatch between the docstring and actual keyword name

Reply:
See below and latest push